index.html

```diff
@@ -391,6 +407,10 @@ <h3>Media analysis visualization</h3>
           bounding boxes). A web application then uses this timed metadata
           to overlay labels and annotations on the video using HTML and CSS.
         </p>
+        <p>
+          This use cases requires frame accurate synchronization of the content
```

Comment:
s/use cases/use case

Reply:
Not part of this update, and I cannot comment on parts that didn't change, but there seems to be an extra "the" five lines above this line: s/that describe the when the objects/that describe when the objects

Reply:
:+1: